nix/release.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, lib, fetchFromGitHub, git, gnused }:
+
+let rev = "b5552cc32f567052dc8c5e6e3af84f8a2d254c35";
+    sha256 = "0gws4lajaylpx1vkq8kpq9q8sk0qil993n9y38xxfwvdmr5gj375";
+in
+  stdenv.mkDerivation rec {
+    name = "git-bubbles-${rev}";
+
+    src = fetchFromGitHub {
```

Comment:
Oh I didn’t see that, this could be replaced by `src = ./..`, this way you don’t have to specify a revision

Reply:
Indeed, I wasn't sure about the best strategy as I'm not tagging any specific revision in this project for now.

Reply:
Done, thanks for the tip.